### PR TITLE
added more items to example config

### PR DESCRIPTION
**Pull request by khag7**
## Description
Added some items to the example config that some users may not know about.

## Motivation and Context
With so many command line flags, it is very easy for existing users and new users alike to miss features. Not every flag needs to be in the config file, but it wouldn't hurt either.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: config/config.ini.example [hunk 1]
```diff
@@ -10,20 +10,26 @@
...
+#spawnpoint-only        # does a normal "spiral" step scan but skips any steps with no known spawnpoints (default false)
+#spawnpoint-scanning    # no "spiral" step scan, jumps from spawnpoint to spawnpoint based on the known time that they will appear
+                        #(takes a file path to a json of known spawnpoints, if the file does not exist will use all known points from the database)
 
 # Misc
 #gmaps-key:             # your Google Maps API key
 #proxy:                 # Proxy URL e.g. socks5://127.0.0.1:9050
 #webhook:               # webhook URL (including http://)
 #webhook-updates-only:               # only send updates to webhooks, (excludes gyms & non-lured pokéstops)
+#print-satus:           # shows the status screen instead of log screen, can switch between with (ENTER)
```
Comment (jnovack) [inline config/config.ini.example]: print what now? a satus?

Comment (Becold) [inline config/config.ini.example]: Add `(default false)` like line 18-19-20
And `status` not `satus`

File: config/config.ini.example [hunk 2]
```diff
@@ -29,14 +29,14 @@
...
-#print-satus:           # shows the status screen instead of log screen, can switch between with (ENTER)
+#print-status:           # shows the status screen instead of log screen, can switch between with [ENTER] (default false)
 
 # Webserver settings
 #host:                  # address to listen on (default 127.0.0.1)
 #port:                  # port to listen on (default 5000)
 #locale:                # pokemon translation
-#ssl-certificate:       # path to ssl certificate
-#ssl-privatekey:        # path to ssl private key
+#ssl-certificate:       # relative path to ssl certificate (e.g. ./config/ssl.cert)
```
Comment (Becold) [inline config/config.ini.example]: This path can be absolute too. (Proof: I did it)
But keep the e.g.

File: config/config.ini.example [hunk 3]
```diff
@@ -29,14 +29,14 @@
...
+#ssl-privatekey:        # relative path to ssl private key (e.g. ./config/ssl.key)
```
Comment (Becold) [inline config/config.ini.example]: same as above

File: config/config.ini.example [hunk 4]
```diff
@@ -29,14 +29,14 @@
 #proxy:                 # Proxy URL e.g. socks5://127.0.0.1:9050
 #webhook:               # webhook URL (including http://)
 #webhook-updates-only:               # only send updates to webhooks, (excludes gyms & non-lured pokéstops)
-#print-satus:           # shows the status screen instead of log screen, can switch between with (ENTER)
+#print-status:           # shows the status screen instead of log screen, can switch between with [ENTER] (default false)
 
 # Webserver settings
 #host:                  # address to listen on (default 127.0.0.1)
 #port:                  # port to listen on (default 5000)
 #locale:                # pokemon translation
```
Comment (Becold) [inline config/config.ini.example]: Please add `(default: en)`

File: config/config.ini.example [hunk 5]
```diff
@@ -10,27 +10,33 @@
...
 #db-port:               # default 3306
+#db-threads:            # Number of db threads; increase if the db queue falls behind. (default 1)
 
 # Search settings
 #location:
+#fixed-location:        # prevents location from being changed (default false)
 #no-gyms:               # disables gym scanning (default false)
 #no-pokemon:            # disables pokemon scanning (default false)
 #no-pokestops:          # disables pokestop scanning (default false)
 #scan-delay:            # default 10
```
Comment (Becold) [inline config/config.ini.example]: Please explain what it does + default inside ( )

File: config/config.ini.example [hunk 6]
```diff
@@ -10,27 +10,33 @@
...
 #step-limit:            # default 12
```
Comment (Becold) [inline config/config.ini.example]: Same as above

**Comment by darkelement1987** [comment]
Even more commands here:

https://github.com/PokemonGoMap/PokemonGo-Map/wiki/Command-Line-Arguments

**Comment by jnovack** [comment]
jitter

File: config/config.ini.example [hunk 7]
```diff
@@ -10,27 +10,34 @@
...
+#fixed-location:        # prevents location from being changed (default false)
 #no-gyms:               # disables gym scanning (default false)
 #no-pokemon:            # disables pokemon scanning (default false)
 #no-pokestops:          # disables pokestop scanning (default false)
-#scan-delay:            # default 10
-#step-limit:            # default 12
+#scan-delay:            # seconds to wait before sending an account to do its next scan (default 10)
+#step-limit:            # search radius measured as 140m "steps" from the current search location (default 12)
+#jitter:                # add a small amount of random variability to step locations in an effort to appear less automated (default false)        
+#spawnpoint-only:       # does a normal "spiral" step scan but skips any steps with no known spawnpoints (default false)
```
Comment (jabberjabber) [inline config/config.ini.example]: Pretty sure this has been changed to --skip-empty

Comment (beccasafan) [inline config/config.ini.example]: Yeah this is pretty out-of-date. Needs a rebase and current info, or just a new PR and we'll close this.

**Comment by primaxius** [comment]
Planning on a full re-write of the config.ini.example shortly and this is out of date

**Pull request closed without merging**